projects/fsastorefrontlib/src/core/my-account/connectors/consent.adapter.ts

```diff
@@ -97,4 +97,19 @@ export abstract class ConsentAdapter {
     oboCustomerId: string,
     address: Address
   ): Observable<{}>;
+
+  /**
+   * Abstract method used to update permissions given to Consent Holder
+   *
+   * @param userId The `userId` identifier of the consent holder
+   * @param oboConsentHolderUid `Uid` of customer created by consent holder for which permission should be updated.
```

Comment:
'Uid' of consent holder for which permission should be updated.

Reply:
I copied it from swagger 🙂

Reply:
Then we have one more place to fix it. Thanks :)